conans/client/tools/net.py

```diff
@@ -3,17 +3,25 @@
 import os
 from conans.client.output import ConanOutput
 from conans.client.rest.uploader_downloader import Downloader
-from conans.client.tools.files import unzip
+from conans.client.tools.files import unzip, check_md5, check_sha1, check_sha256
 from conans.errors import ConanException
 
 _global_requester = None
 
 
-def get(url):
-    """ high level downloader + unziper + delete temporary zip
+def get(url, md5='', sha1='', sha256=''):
```

Comment:
Thanks very much for your contribution!

However, I feel the interface could be slightly improved, because current one allows defining more than one hash, but only one can be checked. I would go for a ``get(url, checksum=None)`` method, raising an Error if checksum is an invalid value. Allowed values would be strings "md5", etc. This allows also for extensibility without changing the method signature.

Reply:
@memsharded how the function would look like? you need two parameters then, one for the algorithm and other for the value, right?

Reply:
Oh, yes, my bad, sorry. I didn't read properly the code.
In that case, I think I prefer the original proposal, with just one parameter. Two parameters like ``get(url, checksum="md5", value="xxxx")`` doesn`t read better than ``get(url, md5="xxx")``

Reply:
Thanks for the feedback.  I do like the simplicity of having to only use a single hash argument, but I understand the concern about the consumer passing multiple hash arguments and getting unexpected behavior.  Other options:

1. Assert that only a single hash argument is provided.
2. Change the `elif`s to `if`s, so if multiple hash arguments are provided, they're all checked.

Preference?

Reply:
Agree with the simplicity of 1 parameter for user.

My preference is 2) change the elifs to ifs. It is a bit weird, but several checksums can indeed be checked for super-mega-ultra security :)

Reply:
OK, I just pushed that change.